test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_Thread.java

```diff
@@ -787,10 +786,10 @@ public void test_join() {
 			// }
 			th.join();
 		} catch (InterruptedException e) {
-			result = false;
+			// killer.interrupt();
+			AssertJUnit.fail("Hung joining a non-started thread", e);
 		}
 		// killer.interrupt();
```

Comment:
What is the purpose of the comments referring to 'killer' (there is nothing available here by that name)?

Reply:
I couldn't say, just keeping it consistent with what was already there. 
I've removed the hopefully obsolete comments.